Fabric/src/main/java/mezz/jei/fabric/JustEnoughItemsClient.java

```diff
@@ -23,6 +25,11 @@ public void onInitializeClient() {
 		Translator.setLocaleSupplier(new MinecraftLocaleSupplier());
 		ClientLifecycleHandler clientLifecycleHandler = new ClientLifecycleHandler();
 
+		ClientRecipeSynchronizedEvent.EVENT.register((minecraft, synchronizedRecipes) -> {
+            Internal.setClientSyncedRecipes(RecipeMap.create(synchronizedRecipes.recipes()));
```

Comment:
For the future, it may be nice if the event provided a shared `RecipeMap`, to avoid having multiple mods create it